vtex/utils/transform.ts

```diff
@@ -463,7 +463,30 @@ const toBreadcrumbList = (
   { baseUrl }: ProductOptions,
 ): BreadcrumbList => {
   const { categories, productName } = product;
-  const names = categories[0]?.split("/").filter(Boolean);
+
+  // Prevents malformed breadcrumbs when a category name contains slashes
+  const names = categories.toReversed().reduce((acc, path, index) => {
+    // Remove barras do início e do fim do caminho, se houver
+    const trimmedPath = path.startsWith("/") ? path.slice(1) : path;
+    const finalPath = trimmedPath.endsWith("/")
+      ? trimmedPath.slice(0, -1)
+      : trimmedPath;
+
+    // Se for a primeira iteração, apenas adicione o caminho ao acumulador
+    if (index === 0) return [finalPath];
+
+    // Obtenha o último item do acumulador para comparação
+    const lastItem = acc[index - 1];
+
+    // Extraia a parte do caminho que vem após o último item
+    const newItem = finalPath.includes(lastItem)
+      ? finalPath.split(lastItem)[1]
+      : finalPath;
```

Comment:
@AlanVncs there is an issue with the last split:

<img width="512" alt="image" src="https://github.com/user-attachments/assets/61fad82a-679a-4b53-8fa6-eae60899c180">